conans/client/generators/cmake_common.py

```diff
@@ -698,12 +710,19 @@ def _conan_basic_setup_common(addtional_macros, cmake_multi=False):
 
 macro(conan_set_find_paths)
     if(CMAKE_BUILD_TYPE)
+        MESSAGE("BUILD TYPE: ${CMAKE_BUILD_TYPE}")
```

Comment:
_This variable is only meaningful to single-configuration generators (such as make and Ninja) i.e. those which choose a single configuration when CMake runs to generate a build tree as opposed to multi-configuration generators which offer selection of the build configuration within the generated build environment_

Will this work for cmake_multi i.e. for CMake MSVC IDE generator?